File: eTierMeeting_v2/API/Models/MachineCheckList/HistoryInventory.cs

using System.ComponentModel.DataAnnotations;

namespace Machine_API.Models.MachineCheckList
{
    public class HistoryInventory
    {
        public int HistoryInventoryID { get; set; }

        public int? InventoryType { get; set; }

        [StringLength(20)]
        public string PlnoID { get; set; }

        public int? CountComplete { get; set; }

        public int? CountWrongPosition { get; set; }

        public int? CountNotScan { get; set; }

        [StringLength(20)]
        public string CreateBy { get; set; }

        public DateTime? StartTimeInventory { get; set; }

        public DateTime? EndTimeInventory { get; set; }

        public DateTime? CreateTime { get; set; }

    }
}

File: eTierMeeting_v2/SPA/src/app/_core/_models/history-inventory.ts
export interface HistoryInventory {
    historyInventoryID: number;
    inventoryType: number | null;
    idPlno: string;
    place: string;
    countComplete: number | null;
    countWrongPosition: number | null;
    countNotScan: number | null;
    userName: string;
    empName: string;
    startTimeInventory: string | null;
    endTimeInventory: string | null;
    dateTime: string | null;
    typeFile: string;
}
